src/popup/components/diagnostic-view-toggle.scss

```diff
@@ -5,7 +5,6 @@
 .diagnostic-view-toggle {
     display: grid;
     grid-template-columns: auto auto;
-    grid-template-rows: 25px 25px;
```

Comment:
Instead of removing the row specification entirely, I'd recommend updating such that each row uses a minimum of the original 25px and a maximum of `min-content`, which will allow it to expand as necessary for content that's too big for 25px, but only as much as necessary. This has better behavior than the default behavior you get by removing `grid-template-rows` entirely for the "Color" case in particular, where it's still one line after using a very large font size and gets unintended spacing applied with the default behavior.

"Very large" font size with no `grid-template-rows`:
![screenshot of popup page with suggested change showing inconsistent margin between "color" title and link](https://user-images.githubusercontent.com/376284/210006985-c60b54e0-4fd6-44e1-a256-97f5f5116e5d.png)

"Very large" font size with `grid-template-rows: minmax(25px, min-content) minmax(25px, min-content);` (note the more consistent spacing between the "Color" toggle's title and its "How to test color" link):
![screenshot of popup page with suggested change showing consistent margin between "color" title and link](https://user-images.githubusercontent.com/376284/210007020-e6bec332-2495-4684-9571-436ff1b2e5cb.png)

Reply:
Thanks for the recommendation, I agree it looks more consistent, especially with the "Color" tittle.